include/envoy/runtime/runtime.h

```diff
@@ -22,6 +22,13 @@ class RandomGenerator {
   virtual uint64_t random() PURE;
 
   /**
+   * Writes out_len bytes of random data to out.
+   * @param out supplies output buffer.
+   * @param out_len supplies number of requested bytes / length of the output buffer.
+   */
+  virtual void random_bytes(uint8_t* out, size_t out_len) PURE;
```

Comment:
Sure, my thinking here was that it's pretty generic interface for randomness for any 3rd-party filters that would like to use it.